pom.xml

```diff
@@ -99,16 +99,43 @@
         <groupId>org.apache.maven</groupId>
         <artifactId>maven-core</artifactId>
         <version>${maven.version}</version>
+        <scope>provided</scope>
       </dependency>
       <dependency>
         <groupId>org.apache.maven</groupId>
         <artifactId>maven-model</artifactId>
         <version>${maven.version}</version>
+        <scope>provided</scope>
       </dependency>
       <dependency>
         <groupId>org.apache.maven</groupId>
         <artifactId>maven-plugin-api</artifactId>
         <version>${maven.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.maven</groupId>
+        <artifactId>maven-artifact</artifactId>
+        <version>${maven.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
```

Comment:
@hboutemy can you explain why we need these dependencies ?

Reply:
great question
given Maven dependency conflict resolution algorithm, the dependencies I added were not automatically found from provided transitives but as transitives from compile scope: adding explicitely as dependencyManagement marks them provided as I wanted in this PR
notice: seeing that type of fact is one really nice feature of m2e in Eclipse